src/EditorFeatures/Test/EditAndContinue/ActiveStatementDescription.cs

```diff
@@ -112,7 +112,8 @@ internal static int[] GetIds(string ids)
                 var ids = GetIds(match.Groups["Id"].Value);
                 int absoluteOffset = offset + markedSyntax.Index;
 
-                yield return ValueTuple.Create(new TextSpan(absoluteOffset, markedSyntax.Length), ids);
+                var span = markedSyntax.Length != 0 ? new TextSpan(absoluteOffset, markedSyntax.Length) : new TextSpan();
```

Comment:
The misunderstanding was that I thought this was a product change.  This just corrects a test helper.